### ISSUE TITLE
Using variables in descriptions

### ISSUE THREAD

**Issue by afdaniele** [fabiospampinato/vscode-terminals#71]
Hi, I noticed that terminals created with this extension do not support descriptions containing variables.
For example, I set the terminal description to be the following,

![image](https://user-images.githubusercontent.com/22665009/177644954-ba301f46-9bdc-4845-83c2-a640be7f36de.png)

Then I create two terminals, one called `terminal1` without using the extension and another one called `terminal2` using the extension instead. This is what I get,

![image](https://user-images.githubusercontent.com/22665009/177645074-3b738a3d-6b33-42cc-9641-1506117d7a55.png)

Is this a known problem?, thank you.

**Comment by fabiospampinato** [comment]
Yeah I guess I didn't think about implementing this, it should probably be implemented.

**Comment by fabiospampinato** [comment]
What configuration are you using to get the description to show up in the tab like that?

**Comment by fabiospampinato** [comment]
This is supported as of v2.0.1 👍
